src/table-utils/utils.ts

```diff
@@ -28,3 +30,19 @@ export const getTableDimensions = (node: PmNode | Node) => {
 
     return {rows: rows || 1, cols: cols || 1};
 };
+
+export function atEndOfCell(view: EditorView, dir: number) {
+    if (!(view.state.selection instanceof TextSelection)) return null;
```

Comment:
you should use `isTextSelection` helper from `src/utils/selection` instead of `instanceof`